src/App.vue

```diff
@@ -1158,9 +1158,7 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
 
   /** Fetches and stores business info. */
   private async loadBusinessInfo (businessId: string): Promise<void> {
-    const response = await LegalServices.fetchBusinessInfo(businessId)
-
-    const business = response?.data?.business as BusinessIF
+    const business = await LegalServices.fetchBusinessInfo(businessId).catch(() => {})
```

Comment:
I changed fetchBusinessInfo to actually return the business object, which simplifies this code.

Note that `business` will be undefined if the fetch fails.